### PR TITLE
[LIBWEB-1353] Fix cookie write error

### PR DESCRIPTION
**Pull request by zikaari**
Analytics would fail fatally if cookies are made inaccessible by overriding the cookie getter/setter by a piece of code. This patch adds resilience against such a case.

### CODE REVIEW

**Review by silesky — APPROVED** [review]
LGTM

File: packages/browser/src/core/user/index.ts [hunk 1]
```diff
@@ -62,15 +62,14 @@ const ONE_YEAR = 365
...
 export class Cookie extends Store {
   static available(): boolean {
-    let cookieEnabled = window.navigator.cookieEnabled
-
-    if (!cookieEnabled) {
-      jar.set('ajs:cookies', 'test')
-      cookieEnabled = document.cookie.includes('ajs:cookies')
-      jar.remove('ajs:cookies')
+    try {
+      jar.set('ajs_cookies', 'test')
```
Comment (oscb) [inline packages/browser/src/core/user/index.ts:66]: nit: for quick reuse putting `ajs_cookies` as a const `AJS_COOKIE_AVAILABILITY_KEY` or similar.

I would also suggest renaming it to `ajs_cookies_check` or something to highight that it is only an availability check cookie.

**Review by oscb — APPROVED** [review]

File: packages/browser/src/core/user/index.ts (hunk 1 above)
Comment (silesky) [reply packages/browser/src/core/user/index.ts:66]: good point! All of these strings get repeated in the bundle -- using a constant saves bytes!

Comment (zikaari) [reply packages/browser/src/core/user/index.ts]: Updated

File: .changeset/pink-dancers-exist.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'@segment/analytics-next': minor
```
Comment (chrisradek) [inline .changeset/pink-dancers-exist.md:2]: Nit: Probably a patch since this is a bug fix/doesn't introduce any new APIs.

**Review by chrisradek — APPROVED** [review]
Looks good to me! Just minor comment on versioning.

**Pull request merged**